tests/test_scalars.py

```diff
@@ -745,8 +746,12 @@ def test_timedelta_mul_div() -> None:
         md_ndarray_float // td  # type: ignore[operator] # pyright: ignore[reportGeneralTypeIssues]
         mp_series_int // td  # type: ignore[operator] # pyright: ignore[reportGeneralTypeIssues]
         md_series_float // td  # type: ignore[operator] # pyright: ignore[reportGeneralTypeIssues]
-        md_int64_index // td  # type: ignore[operator] # pyright: ignore[reportGeneralTypeIssues]
-        md_float_index // td  # type: ignore[operator] # pyright: ignore[reportGeneralTypeIssues]
+        assert_type(
+            md_int64_index // td, Never  # pyright: ignore[reportGeneralTypeIssues]
+        )
+        assert_type(
+            md_float_index // td, Never  # pyright: ignore[reportGeneralTypeIssues]
+        )
```

Comment:
Concerned about the above change.  Previously, `mypy` was seeing that this is an invalid operation.  If you make the above change, mypy is no longer detecting that.  That's not a good thing.

Reply:
I agree, I don't know how to fix it. I think the reason why it worked previously was a behavior of Never when it is being used on input arguments: it is interpreted by mypy/pyright to indicate that that call is invalid. Now, with the generic class, we need to have annotations on the input arguments (we don't have the luxury of using Never there anymore). We can still return Never (but that has a slightly different semantic meaning).

Reply:
I found a fix.  I created a PR against your branch at https://github.com/twoertwein/pandas-stubs/pull/3

Key was to remove `__floordiv__()` from `OpsMixin` and let any subclass of `OpsMixin` define only the valid values.

Reply:
So the mypy folks say it's not a bug, and pyright says that its implementation was incorrect. So I think the only way to do this is with what I did - you can't use `Never` or `NoReturn` to "override" the implementation in a subclass that matches `Any`